source/exercises100.ktx

```diff
@@ -855,22 +855,12 @@ F = np.bincount(I,X)
 print(F)
 
 < q66
-Considering a (w,h,3) image of (dtype=ubyte), compute the number of unique colors (★★★)
+Considering a (w,h,3) image of (dtype=ubyte), compute the number of unique colors (★★☆)
 
 < h66
 hint: np.unique
 
 < a66
-# Author: Nadav Horesh
-
-w,h = 16,16
-I = np.random.randint(0,2,(h,w,3)).astype(np.ubyte)
-# The parenthesis around 256*256 ensure the dtpye promoted to uint32 properly.Otherwise the first term will overflow.
-F = I[...,0]*(256*256) + I[...,1]*256 +I[...,2]
-n = len(np.unique(F))
-print(np.unique(I))
-
-# Another solution
 # Author: Fisher Wang
 
 w, h = 16, 16
```

Comment:
w, h = 256, 256
img = np.random.randint(0, 4 (w, h, 3)).astype(np.ubyte)

This wil "minimize" the number of unique colors. If you use 256 values for a 16x16 image, chances are that all colos are unique.